client/src/components/navbar/Navbar.js

```diff
@@ -1,29 +1,55 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+import { Link, Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { Button } from 'semantic-ui-react';
 
 import '../../css/Navbar.css';
-
 import ProfileDropdown from '../ProfileDropdown/ProfileDropdown';
+import { levelEnum } from '../../utils/consts';
 import * as Routes from '../../routes';
 
 /**
  * Navbar display to view user sesion and React-route-dom navigation
  * @param {Object} user the current user of the session
  */
-const Navbar = ({ user }) => (
-  <nav className="nav">
-    <h2 id="nav-title">
-      <Link className="nav-link" to={Routes.DEFAULT}>
-        H4I Member Database
-      </Link>
-    </h2>
-    <div className="profile-item">
-      <h2 id="welcome-text">Hello, {user.firstName}!</h2>
-      <ProfileDropdown user={user} />
-    </div>
-  </nav>
-);
+const Navbar = ({ user }) => {
+  const [redirectToNewMemberPage, setRedirectToNewMemberPage] = useState(false);
```

Comment:
Maybe we check the route and redirect conditionally based on that instead of introducing new state?